### ISSUE TITLE
TypeError: cannot unpack non-iterable AudioMetaData object

### ISSUE THREAD

**Issue by 0xd3ba** [facebookresearch/denoiser#68]
Was trying out in Google Colab and encountered this when I tried to enhance using `!python3 -m denoiser.enhance --dns48 --noisy_dir=/content/MS-SNSD/NoisySpeech_training --out_dir=/content/dns48_outputs`

```
INFO:denoiser.pretrained:Loading pre-trained real time H=48 model trained on DNS.
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.7/dist-packages/denoiser/enhance.py", line 138, in <module>
    enhance(args, local_out_dir=args.out_dir)
  File "/usr/local/lib/python3.7/dist-packages/denoiser/enhance.py", line 114, in enhance
    dset = get_dataset(args)
  File "/usr/local/lib/python3.7/dist-packages/denoiser/enhance.py", line 95, in get_dataset
    files = find_audio_files(paths.noisy_dir)
  File "/usr/local/lib/python3.7/dist-packages/denoiser/audio.py", line 27, in find_audio_files
    siginfo, _ = torchaudio.info(file)
TypeError: cannot unpack non-iterable AudioMetaData object
```

Similar error when I tried `!python3 -m denoiser.audio MS-SNSD/NoisySpeech_training > noisy.json` as well.

**Comment by 0xd3ba** [comment]
**Update:**  The root cause is because I installed it with `pip`.  Cloning the repo and doing the same didn't cause any issue. I guess the package in PyPI needs an update.

**Comment by adefossez** [comment]
Yes good point, I did not update the package on pip, I will do that today, thank you for reporting.

**Comment by adefossez** [comment]
This should be fixed now with v0.1.4. https://pypi.org/project/denoiser/0.1.4/
